partcad/src/partcad/test/cam.py

```
@@ -0,0 +1,117 @@
+            return True
+
+        if not shape.is_manufacturable and not "force_manufacturing" in test_ctx:
+            self.debug(shape, "Not supposed to be manufacturable")
+            return True
+
+        if isinstance(shape, Part):
+            return await self.test_part(tests_to_run, ctx, shape, test_ctx)
+        elif isinstance(shape, Assembly):
+            return await self.test_assembly(tests_to_run, ctx, shape, test_ctx)
+
+    async def test_part(self, tests_to_run: list[Test], ctx, part: Part, test_ctx: dict = {}) -> bool:
```

Ruff (0.8.2)

Unused method argument: `tests_to_run`

(ARG002)